site/content/blog/2021-02-03-run-zap-without-java-using-docker-and-webswing/index.md

```diff
@@ -51,15 +51,15 @@ Note that on Windows you will need to replace `$(pwd)` with the full current wor
 
 When you do this ZAP will create 2 files on your mapped drive:
 
-* `owasp_zap_root_ca.crt` - the public ZAP Root CA certificate
-* `owasp_zap_root_ca.key` - the private ZAP Root CA certificate
+* `zap_root_ca.crt` - the public ZAP Root CA certificate
+* `zap_root_ca.key` - the private ZAP Root CA certificate
```

Comment:
Even though we havnt release 2.14?
I think the website should be correct for the last full release..

Reply:
Oh! This is no longer in the core 😛

Reply:
Not yet, by some hours. I guess it will have to wait for next week's release.